site/source/user-manual/service-guides/vaultwarden/bitwarden-walkthrough.rst

```diff
@@ -0,0 +1,457 @@
+                    :alt: vaultwarden-login-email
+
+    #. Then your master password:
+
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-login-password.png
+                    :width: 50%
+                    :alt: vaultwarden-login-password
+
+    #. We are greeted with our newly setup Vaultwarden password vault!
+
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-fresh-vault.png
+                    :width: 50%
+                    :alt: vaultwarden-fresh-vault
+    
+    #. At the top you can visit **Tools** for reports and to use the password generator. You can also import data from another program, such as lastpass, onepass, or keepass!
+
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-tools.png
+                    :width: 50%
+                    :alt: vaultwarden-tools
+    
+    #. **Account settings** gives you all your options, including the ability to set up 2 factor authorization.
+
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-account-settings.png
+                    :width: 50%
+                    :alt: vaultwarden-account-settings
+
+    #. Now you're ready to start storing some passwords!
+
+.. collapse:: Storing your first password
+
+    #. Back at the main page for your vault, let's do a quick example login. We'll start by creating a folder for it:
+
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-click-add-folder.png
+                    :width: 50%
+                    :alt: vaultwarden-click-add-folder
+    
+    #. Name the folder and click **Save**:
+
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-add-folder.png
+                    :width: 50%
+                    :alt: vaultwarden-add-folder
+
+    #. Click **Add item**:
+
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-add-item.png
+                    :width: 50%
+                    :alt: vaultwarden-add-item
+
+    #. Now add the credentials and click **Save**:
+
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-add-new-login.png
+                    :width: 50%
+                    :alt: vaultwarden-add-new-login
+
+    #. And there we have it - our first set of login credentials.
+
+        .. figure:: /_static/images/services/vaultwarden/vaultwarden-new-login.png
+                    :width: 50%
+                    :alt: vaultwarden-new-login
+
+    Congratulations! You have setup your own self-hosted password manager and have added a set of login credentials to it!
+
+
+
+    .. _vaultwarden-admin:
+
+.. collapse:: Admin Console
+
+    Accessing the Admin Console
+    ===========================
+    You will find the Admin console by adding ``/admin`` to the end of your Vaultwarden WebUI URL, found in the "Interfaces" section of the Vaultwarden Service page.  You will be prompted for your "Admin Token," which is also found on your Vaultwarden Service page under "Properties."
```

Comment:
Maybe a screenshot, and/or a quick note about what you would want to use this for.  It may also be worth mentioning what you cannot do here - iirc there is no ability to reset passwords for users here